packages/backend/app.js

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env node
 import express from "express";
-import apiRouter from "./webui/api.js";
+import apiRouter from "../../webui/api.js";
```

Comment:
The api.js should be okay to move in this folder without it causing side-effects on the frontend package.